### PR TITLE
feat: support Example section for google docstring

### PR DESCRIPTION
**Pull request by gfyts0123**

### CODE REVIEW

**Review by rr- — CHANGES_REQUESTED** [review]
Thank you for the pull request.

- Let's rename DocstringExamples to DocstringExample
- Let's add missing tests
- Please run black
- Please change `var # type hint` to a `var: TypeHint`

**Comment by gfyts0123** [comment]
Thank you for the reply, I have submitted it again.
If there are still any questions, please let me know.

File: docstring_parser/common.py [hunk 1]
```diff
@@ -184,3 +198,9 @@ def deprecation(self) -> T.Optional[DocstringDeprecated]:
             if isinstance(item, DocstringDeprecated):
                 return item
         return None
+
+    @property
+    def examples(self) -> T.Optional[DocstringDeprecated]:
```
Comment (rr-) [inline docstring_parser/common.py:203]: `T.List[DocstringExample]`

**Review by rr- — CHANGES_REQUESTED** [review]
Thanks for the changes. A couple of comments left:

Let's remove the assertions on `docstring.meta`.
Let's change the assertions on `test_empty_example` from `docstring.meta` to `docstring.examples`.

**Comment by gfyts0123** [comment]
Thanks for the reply, I've changed it.
If there are still any questions, please let me know.

**Comment by rr-** [comment]
It's still `T.Optional`, whereas the function returns a list :) Please update it to `T.List` and we should be good to go.

**Review by rr- — CHANGES_REQUESTED** [review]
^

**Comment by gfyts0123** [comment]
Oh I found it, thanks for figuring it out, it should be OK now.

**Review by rr- — APPROVED** [review]

**Comment by rr-** [comment]
Merged manually. Thanks.

**Pull request closed without merging**